### PR TITLE
migrate from travis-ci to github actions

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

**Comment by jhoblitt** [comment]
@andylytical I am trying to resurrect CI by migrating to github actions.  Recent versions of `yamllint` are now paying attention to quoting for array values, which required fixing up some of the NCSA hierarchy.  Of course, github actions are having trouble today...

File: hieradata/org/ncsa.yaml [hunk 1]
```diff
@@ -404,84 +404,84 @@ unbound::backup_dns_servers:
...
+  - "[23.172.in-addr.arpa., 130.126.2.131]"
+  - "[10.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[24.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[25.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[26.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[27.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[28.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[29.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[30.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[31.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
```
Comment (andylytical) [inline hieradata/org/ncsa.yaml:487]: Each line is an arrays of strings, not one single string.
correct format should be: `- ["22.172.in-addr.arpa.", "130.126.2.131"]`

**Review by andylytical — CHANGES_REQUESTED** [review]
@jhoblitt One issue found in file `hieradata/org/ncsa.yaml` at section `unbound::reverse_overrides:` lines 407-487

File: hieradata/org/ncsa.yaml (hunk 1 above)
Comment (jhoblitt) [reply hieradata/org/ncsa.yaml:487]: Ugh. That would have been bad...

Comment (andylytical) [reply hieradata/org/ncsa.yaml:487]: See also: https://github.com/lsst-it/puppet-unbound/blob/master/manifests/init.pp

Comment (jhoblitt) [reply hieradata/org/ncsa.yaml]: I did the conservative approach and simply disabled that lint rule for that section.

**Review by andylytical — APPROVED** [review]

File: hieradata/org/ncsa.yaml (hunk 1 above)
Comment (andylytical) [reply hieradata/org/ncsa.yaml]: Looks good! Thanks!

**Pull request merged**